src/go/k8s/pkg/resources/pdb.go

```diff
@@ -16,7 +16,7 @@ import (
 	"github.com/go-logr/logr"
 	redpandav1alpha1 "github.com/redpanda-data/redpanda/src/go/k8s/apis/redpanda/v1alpha1"
 	"github.com/redpanda-data/redpanda/src/go/k8s/pkg/labels"
-	policyv1beta1 "k8s.io/api/policy/v1beta1"
```

Comment:
Looks good, have you tested an upgrade on existing the budgets?  Deprecation guide: https://kubernetes.io/docs/reference/using-api/deprecation-guide/

Reply:
I didn't test that. 

The description from the docs

### The policy/v1beta1 API version of PodDisruptionBudget is no longer served as of v1.25.

* Migrate manifests and API clients to use the policy/v1 API version, available since v1.21.
* All existing persisted objects are accessible via the new API
* Notable changes in policy/v1:
  * an empty `spec.selector` (`{}`) written to a `policy/v1` PodDisruptionBudget selects all pods in the namespace (in `policy/v1beta1` an empty `spec.selector` selected no pods). An unset `spec.selector` selects no pods in either API version.

From my point of view nothing will break from the functionality perspective.

Never the less I will use our cloud product to test this.

Reply:
yep, doesnt seem there is much for us to do, but worth checking. I did see our operator fail for this version, so approving this and will trust you will rebase